src/test/unit/com/hltech/pact/gen/domain/pact/PactFactoryUT.groovy

```diff
@@ -264,6 +249,26 @@ class PactFactoryUT extends Specification {
             }
     }
 
+    def "should get media type from feign client"() {
+        when:
+            final Pact pact = pactFactory.createFromFeignClient(MediaTypeFeignClient, 'SpecConsumer', objectMapper)
```

Comment:
Why we need final here?

Reply:
I just copied from above. It doesn't matter, especially in tests.

Reply:
So it can be removed ;)

Reply:
yeah